CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+3.x.x
+------
+
+**ENHANCEMENTS**
+- Limit privileges associated with IAM Policies created within the cluster.
```

Comment:
To review

Reply:
Removed: https://github.com/aws/aws-parallelcluster/pull/3678/commits/c869fcec4d69c87246ab2923e0089e5ab0c8f569